[LTC] Fix linter issues for lazy_tensor_core subdirectory

alanwaketan opened the pull request:
Summary:
This patch fixes linter issues for lazy_tensor_core subdirectory.

Fixes #63109.

Commit:
[LTC] Fix linter issues for lazy_tensor_core subdirectory

Summary:
This patch fixes linter issues for lazy_tensor_core subdirectory.

alanwaketan commented:
A lot of flake8 issues are related to indentation size. Therefore, will upload a second commit to make the review process easier.

Commit:
[LTC] Fix linter issues for lazy_tensor_core subdirectory

Summary:
This is the second commit that applies autopep8 to the entire subdirectory.

Commit:
[LTC] Fix linter issues for lazy_tensor_core subdirectory

Summary:
Applies more autopep8.

asuhan reviewed (commented):
I like this, pytorch/xla should do it as well.

alanwaketan commented:
> I like this, pytorch/xla should do it as well.

Thanks, Alex. Still working on fixing the remaining linter issues which needs some manual works.

Commit:
[LTC] Fix linter issues for lazy_tensor_core subdirectory

Summary:
Manually fixes some linter issues that don't require too many code changes.

Commit:
[LTC] Fix linter issues for lazy_tensor_core subdirectory

Summary:
Manually fixes more linter issues that require slightly code rewrites.

Commit:
[LTC] Fix linter issues for lazy_tensor_core subdirectory

Summary:
Manually fixes more linter issues that require some code rewrites which
might break things.

alanwaketan commented:
It looks like the linters are happy now. However, the patch seems introducing some breakages. I will dig into them.

desertfire commented:
> It looks like the linters are happy now. However, the patch seems introducing some breakages. I will dig into them.

If it takes time to fix everything, you can always break it into several smaller PRs.

wconstab commented on lazy_tensor_core/lazy_tensor_core/__init__.py:
```diff
@@ -8,90 +13,84 @@
+    os.remove(_tmp_fname)
 
 
 # These needs to be called before the _LAZYC module is loaded.
```
it looks like you moved _LAZYC import above the _setup functions, but according to the comment the _setup functions need to run before _LAZYC import.  I'm not sure what the reasons are for this or of it's still important, but could you confirm?  It might be safer to just mark these as #noqa or whatever the equivalent marking is to suppress linter warnings here

alanwaketan replied:
It was done by autopep8. I'm actually not sure if flake8 would complain. Let me fix it.

alanwaketan commented:
Let's see if the CI failures can be fixed on #63439.

Commit:
[LTC] Fix linter issues for lazy_tensor_core subdirectory

Summary:
Adopted some comments.

alanwaketan commented:
Have ran the following tests locally, and haven't found any issues.
1. [.../pytorch/lazy_tensor_core] test/cpp/build/test_ptltc.
2. [.../pytorch/lazy_tensor_core] python example.py.

Should be safe to land.

alanwaketan commented:
Thanks @wconstab for approving this patch.

alanwaketan commented:
The failures in CI should not be related. Land the commits as they are.
